internal/prices/prices.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/tidwall/gjson"
 )
 
-func PopulatePrices(cfg *config.Config, project *schema.Project) error {
+func PopulatePrices(cfg *config.RunContext, project *schema.Project) error {
```

Comment:
Same comment about the name.
```suggestion
func PopulatePrices(ctx *config.RunContext, project *schema.Project) error {
```